src/main/resources/windows/why-ini.vtl

```diff
@@ -0,0 +1,12 @@
+mainclass=$info.mainClass
+classpath=$info.jarName
+#foreach ($cp in $info.classpaths)
+classpath=;$!{cp}
+#end
+#if ($info.bundleJre)
+jvm_path=${info.jreDirectoryName}
+#end
```

Comment:
This isn't really needed, when not specified Why searches the current directory for Java, the depth is enough that it can find what JP bundles.